### PR TITLE
Improved StudentConnect fields and identifier

### PR DESCRIPTION
**Pull request by wnchan**
Edited PersonTest.java class too, it passes now.

### CODE REVIEW

File: src/test/java/seedu/address/model/person/PersonTest.java [hunk 1]
```diff
@@ -3,13 +3,7 @@
...
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
-import static seedu.address.logic.commands.CommandTestUtil.VALID_DESCRIPTION_BOB;
-import static seedu.address.logic.commands.CommandTestUtil.VALID_EMAIL_BOB;
-import static seedu.address.logic.commands.CommandTestUtil.VALID_MAJOR_BOB;
-import static seedu.address.logic.commands.CommandTestUtil.VALID_NAME_BOB;
-import static seedu.address.logic.commands.CommandTestUtil.VALID_SM_GITHUB_BOB;
-import static seedu.address.logic.commands.CommandTestUtil.VALID_SM_LINKEDIN_BOB;
-import static seedu.address.logic.commands.CommandTestUtil.VALID_YEAR_BOB;
+import static seedu.address.logic.commands.CommandTestUtil.*;
```
Comment (PearlynnT) [inline src/test/java/seedu/address/model/person/PersonTest.java:6]: Wildcard import

Comment (Bearypop) [inline src/test/java/seedu/address/model/person/PersonTest.java]: I think the * import seems to be causing an issue with the checkstyle, might need to change it

Comment (wnchan) [reply src/test/java/seedu/address/model/person/PersonTest.java]: Just fixed, thanks!

**Review by Bearypop — COMMENTED** [review]
Other than the * import, everything else looks good.

**Review by neyapraveen — COMMENTED** [review]
LGTM! Tests seem to work

File: src/test/java/seedu/address/model/person/PersonTest.java (hunk 1 above)
Comment (wnchan) [reply src/test/java/seedu/address/model/person/PersonTest.java]: alright, just fixed it. thanks!

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 2]
```diff
@@ -51,7 +51,8 @@ public class EditCommand extends Command {
 
     public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Details edited successfully! Edited Person: %1$s";
```
Comment (maj0-0) [inline src/main/java/seedu/address/logic/commands/EditCommand.java:52]: Can change "Person" to "Student" for consistency throughout

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 3]
```diff
@@ -51,7 +51,8 @@ public class EditCommand extends Command {
...
     public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in StudentConnect.";
+    public static final String MESSAGE_DUPLICATE_PERSON = "This student is already on StudentConnect as this "
+            + "email has already been used.";
     public static final String MESSAGE_EMAIL_NOT_FOUND = "Person with the provided email not found.";
```
Comment (maj0-0) [inline src/main/java/seedu/address/logic/commands/EditCommand.java:56]: Can change "Person" to "Student" for consistency throughout

**Review by maj0-0 — COMMENTED** [review]
Aside from wording (person -> student), LGTM!

**Pull request merged**